internal/server/routes.go

```diff
@@ -215,12 +216,20 @@ func wrapRoute[Req, Res any](a *API, routeID routeIdentifier, route route[Req, R
 			return err
 		}
 
+		org := getRequestContext(c).Authenticated.Organization
 		if !route.noOrgRequired {
-			if org := getRequestContext(c).Authenticated.Organization; org == nil {
+			if org == nil {
 				return internal.ErrBadRequest
 			}
 		}
 
+		if org != nil {
+			// TODO: limit should be a per-organization setting
+			if err := redis.RateOK(a.server.redis, org.ID.String(), 1000); err != nil {
```

Comment:
Are we going to hit this fast? 1000 requests/min is ~ 16 requests per second, and I could see a mid/largish org hitting that fairly quickly.

Reply:
That's a good point. Raising it to 3000 for now